docs/deployment/feature_gates.md

```diff
@@ -36,6 +36,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootManagedIssuer                 | `false` | `Alpha` | `1.93` |        |
 | VPAForETCD                         | `false` | `Alpha` | `1.94` |        |
 | VPAAndHPAForAPIServer              | `false` | `Alpha` | `1.95` |        |
+| WorkerPoolHashWithNodeAgentSecret  | `false` | `Alpha` | `1.96` |        |
```

Comment:
I find the name of the feature flag hard to understand. Why not call it `NewWorkerPoolHash` or something along those lines? In particular, I would leave the node agent secret out of the feature flag name. After all, the new worker hash is the important part.